src/app.js

```diff
@@ -47,7 +47,9 @@ export class App {
     await this.appUtils.checkUser(this);
   }
 
-  showTT(id) { document.getElementById(id).style.display = 'block'; }
+  showTT(id) {
+    document.getElementById(id).style.display = 'block';
```

Comment:
@Unesdala please fix your editor so that it does not make these autoformat changes, thanks